src/System.Private.CoreLib/shared/System/Environment.Win32OrWin32Pal.cs

```diff
@@ -0,0 +1,168 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace System
+{
+    public static partial class Environment
+    {
+        private static string GetEnvironmentVariableCore(string variable)
+        {
```

Comment:
These may need:

```
#if FEATURE_APPX
            if (ApplicationModel.IsUap)
                return null;
#endif
```

to make the UWP tests happy.

Or maybe just make the environment variables work for UWP as part of this change, update the tests and delete the special casing for them.

Reply:
Yup, I intended to edit this file too but neglected. Thanks.

Reply:
Ok.  It looks like all of GetEnvironmentVariable/SetEnvironmentVariable/GetEnvironmentStrings/FreeEnvironmentVariables are available in UWP now, so I'll just leave it as this has it and then update any tests that fail accordingly.